plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreServerExtension.java

```diff
@@ -158,6 +158,11 @@ public interface PostgreServerExtension {
      * */
     boolean supportsRoleBypassRLS();
 
+    /**
+     * @return true if database supports syntax like "COMMENT ON ROLE roleName IS 'comment'"
+     */
```

Comment:
```suggestion
    /**
     * Determines whether the database supports syntax like {@code COMMENT ON ROLE roleName IS 'comment'} or not
     */
```

Reply:
Looks like something just for the CheckStyle :(

Reply:
It is. In fact, having the summary is better that having a dangling `@return` tag IMO 👀